src/components/Base/BaseSelect.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <SelectRoot v-model="model" :defaultValue :disabled>
+    <SelectTrigger class="SelectTrigger">
+      <SelectValue class="SelectValue" :placeholder />
+      <SelectIcon class="SelectIcon">
+        <!-- 暫定でq-iconを使用 -->
+        <QIcon name="keyboard_arrow_down" size="sm" />
+      </SelectIcon>
+    </SelectTrigger>
+
+    <SelectPortal>
+      <SelectContent
+        class="SelectContent"
+        position="popper"
+        :sideOffset="4"
+        :collisionPadding="8"
+        :arrowPadding="16"
+        avoidCollisions
+        hideWhenDetached
+      >
+        <SelectViewport>
+          <slot />
+        </SelectViewport>
+        <SelectArrow class="SelectArrow" />
+      </SelectContent>
+    </SelectPortal>
+  </SelectRoot>
+</template>
+
+<script setup lang="ts">
+import {
+  SelectContent,
+  SelectPortal,
+  SelectRoot,
+  SelectIcon,
+  SelectArrow,
+  SelectTrigger,
+  SelectValue,
+  SelectViewport,
+} from "radix-vue";
+
+defineProps<{
+  placeholder?: string;
+  defaultValue?: string;
+  disabled?: boolean;
+}>();
+
+const model = defineModel<string>();
+</script>
+
+<style scoped lang="scss">
+@use "@/styles/v2/variables" as vars;
+@use "@/styles/v2/colors" as colors;
+@use "@/styles/v2/mixin" as mixin;
+
+.SelectTrigger {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: vars.$size-control;
+  border-radius: vars.$radius-1;
+  padding-inline: vars.$padding-2 vars.$padding-1;
+  gap: vars.$gap-1;
+  border: 1px solid;
+  text-align: start;
+  transition: background-color vars.$transition-duration;
+  cursor: pointer;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+  color: colors.$display;
+  border-color: colors.$border;
+  background-color: colors.$control;
+
+  &:hover:not(:disabled) {
+    background-color: colors.$control-hovered;
+  }
+
+  &:active:not(:disabled) {
+    background-color: colors.$control-pressed;
+  }
+
+  &:active:not(:disabled) {
+    box-shadow: 0 0 0 transparent;
+  }
+
+  &:focus-visible {
+    @include mixin.on-focus;
+  }
+
+  &[data-placeholder] {
+    color: colors.$display-sub;
+  }
+}
+
+.SelectIcon {
+  color: colors.$display;
+}
+
+:global([data-radix-popper-content-wrapper]) {
+  z-index: vars.$z-index-dropdown !important;
+}
+
+:global(.SelectContent) {
```

Comment:
ここグローバルにしなくていいはずなんですよね。。。。。。
`:deep`で良いはずなんだけど。。。。。。なぜか`Teleport`した`SelectContent`側に`scoped`用の`data-なんとか`が付与されないですね。。。。。。
なぜ`data-*`がつかないのかかなり調べたけどわからなかった。。。

とりあえずradix-vueのバージョンを`1.9.6`まであげると、ここは`:deep`で大丈夫になりました。。。
↓みたいな感じでメモコメント残すか、バージョン上げるのどうでしょう 🙇 🙇 🙇 
```suggestion
// FIXME: radix-vueのバージョンを上げて`:deep`にする
:global(.SelectContent) {
```

Reply:
バージョン上げての確認ありがとうございます…！今後またバージョンが古いことでのバグに遭遇するのも面倒なので、今のうちにバージョンを上げておく選択をしようと思います…！